### ISSUE TITLE
Mysqli extension not enabled

### ISSUE THREAD

**Issue by myoung1620** [awslabs/eb-docker-nginx-proxy#3]
While this is not really an "issue" per se, I'm trying to get this installed with mysqli extension enabled such that my deployment will function with php7 + aws RDS.  Is this possible?  I see that the library is installed but it is not enabled as an extension by default.

**Comment by rushi216** [comment]
yes its possible, u have to create custom docker image with mysqli extension enabled see: https://hub.docker.com/_/php/
